src/api.jl

```diff
@@ -20,7 +20,7 @@ A structure that contains an initial guess for a periodic orbit detection algori
     * `u0` - guess of a point in the periodic orbit
     * `T` - guess of period of the orbit
 """
-struct InitialGuess{U<:AbstractArray{<:Real}, R<:Union{Real, Nothing}}
+struct InitialGuess{U<:AbstractArray{<:Real},R<:Union{Real,Nothing}}
```

Comment:
In this file you removed spaces between types. Why? Keep them, it is more readable with spaces.